src/components/progress-circle/progress-circle.spec.ts

```diff
@@ -74,6 +74,22 @@ describe('MdProgressCircular', () => {
         done();
       });
   });
+
+  it('should clean up the indeterminate animation when the element is destroyed', (done: () => void) => {
```

Comment:
Rather than using `done`, you can wrap the test function in `async()` and it will magically work. Check out the checkbox spec for an example.